manifests/flexvolume-driver/oci-flexvolume-driver.yaml

```diff
@@ -40,7 +40,7 @@ spec:
             secretName: oci-flexvolume-driver
       containers:
         - name: oci-flexvolume-driver
-          image: ghcr.io/oracle/cloud-provider-oci:v1.24.0
+          image: ghcr.io/yashwantgohokar/cloud-provider-oci:v1.25.0
```

Comment:
The repository needs to be oracle

Reply:
Changed in a later commit. This change was made for testing